gpu-operator/life-cycle-policy.rst

```diff
@@ -86,60 +86,59 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
      - Version
 
    * - NVIDIA GPU Operator
-     - v24.3.0
+     - v24.6.0
 
    * - NVIDIA GPU Driver
-     - | `550.90.07 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-90-07/index.html>`_ (recommended),
-       | `550.54.15 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-54-15/index.html>`_ (default),
+     - | `550.90.07 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-90-07/index.html>`_ (default),
        | `535.183.01 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-183-01/index.html>`_,
        | `470.256.02 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-470-256-02/index.html>`_
 
    * - NVIDIA Driver Manager for K8s
-     - `v0.6.8 <https://ngc.nvidia.com/catalog/containers/nvidia:cloud-native:k8s-driver-manager>`__
+     - `v0.6.10 <https://ngc.nvidia.com/catalog/containers/nvidia:cloud-native:k8s-driver-manager>`__
 
    * - NVIDIA Container Toolkit
-     - `1.15.0 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
+     - `1.16.0 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
 
    * - NVIDIA Kubernetes Device Plugin
-     - `0.15.0 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
+     - `0.16.0 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
 
    * - DCGM Exporter
-     - `3.3.5-3.4.1 <https://github.com/NVIDIA/gpu-monitoring-tools/releases>`__
+     - `3.3.6-3.4.2 <https://github.com/NVIDIA/gpu-monitoring-tools/releases>`__
 
    * - Node Feature Discovery
-     - v0.15.4
+     - v0.16.3
 
    * - | NVIDIA GPU Feature Discovery
        | for Kubernetes
-     - `0.15.0 <https://github.com/NVIDIA/gpu-feature-discovery/releases>`__
+     - `0.16.0 <https://github.com/NVIDIA/gpu-feature-discovery/releases>`__
```

Comment:
Note, we converged the device-plugin and gfd repositories. So all future development and releases will occur at the device-plugin repository.
```suggestion
     - `0.16.1 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
```